gtfs/spec/en/reference.md

```diff
@@ -29,174 +29,176 @@ This document explains the types of files that comprise a GTFS transit feed and
-- **Latitude** - The field contains the WGS84 latitude in decimal degrees. The value must be greater than or equal to -90.0 and less than or equal to 90.0. For example: "41.890169" for the Colosseum in Rome.
-- **Longitude** - The field contains the WGS84 longitude in decimal degrees. The value must be greater than or equal to -180.0 and less than or equal to 180.0. For example: "12.492269" for the Colosseum in Rome.
-- **Non-negative Float** - The field contains a non-negative floating point number (which includes 0).
-- **Non-negative Integer** - The field contains a non-negative whole number (which includes 0).
-- **Phone number** - The field contains a phone number.
-- **Date** - The field contains the service day in the YYYYMMDD format. E.g "20180913" for September 13th, 2018. Since time within a service day can be above 24:00:00, a service day often contains information for the subsequent day(s). 
-- **Time** - The field contains the time in the HH:MM:SS format (H:MM:SS is also accepted). For example: "14:30:00" for 2:30PM or "25:35:00" for 1:35AM on the next day. The time is measured from "noon minus 12h" of the service day (effectively midnight except for days on which daylight savings time changes occur). For times occurring after midnight, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. 
-- **Text** - The field contains a string of UTF-8 characters, which is aimed to be displayed and which must therefore be human readable.
-- **Timezone** - The field contains a TZ timezone from the [TZ database](https://www.iana.org/time-zones). Timezone names never contain the space character but may contain an underscore. Refer to http://en.wikipedia.org/wiki/List_of_tz_zones for a list of valid values. For example: "Asia/Tokyo", "America/Los_Angeles" or "Africa/Cairo".
-- **URL** - The field contains a fully qualified URL that includes http:// or https://, and any special characters in the URL must be correctly escaped. See http://www.w3.org/Addressing/URL/4_URI_Recommentations.html for a description of how to create fully qualified URL values.
-
-## Feed Files
+- **Color** - A color encoded as a six-digit hexadecimal number. Refer to [https://htmlcolorcodes.com](https://htmlcolorcodes.com) to generate a valid value (the leading "#" is not included). <br> *Ex. `FFFFFF` for white, `000000` for black or `0039A6` for the A,C,E lines in NYMTA.*
+- **Currency Code** - An ISO 4217 alphabetical currency code. For the list of current currency, refer to [wikipedia](https://en.wikipedia.org/wiki/ISO_4217#Active_codes). <br> *Ex. `CAD` for Canadian dollars, `EUR` for euros or `JPY` for Japanese yen.*
+- **Date** - Service day in the YYYYMMDD format. Since time within a service day can be above 24:00:00, a service day often contains information for the subsequent day(s). <br> *Ex. `20180913` for September 13th, 2018.*
```

Comment:
I suggest we should introduce a separate definition: "Service day"?
Then, the "Date" definition could be referenced in the "Service day" definition.

Reply:
Are you suggesting that we add "Service day" as a field type? Or should it be added as a term definition?

Reply:
Yes. Term definition.

Reply:
My draft definition of "Service day" is below. Thoughts?

>Service day - A span of time during which service on a route operates continuously. A service day may exceed 24:00:00 if service begins on one day and ends on a the following day. For example, service that runs from 08:00:00 on Friday to 01:00:00 am on Saturday, would be denoted as running from 08:00:00 to 25:00:00 on the Friday service day.

Reply:
Avoid making a hard definition, let agencies decided. Specify time roll-over property. Main goal: warn reader of distinction between service and calendar day.